pkg/process/logging_windows.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package process
```

Comment:
shouldn't this have `// +build windows`?

Reply:
Not necessary; the `_windows` part of the file name causes an implicit build constraint on windows.